must-gather/README.md

```diff
@@ -0,0 +1,25 @@
+cluster-logging must-gather
+=================
+
+`cluster-logging-must-gather` is a tool built on top of [OpenShift must-gather](https://github.com/openshift/must-gather)
+that expands its capabilities to gather Openshift Cluster Logging information.
+
+### Usage
+```sh
+oc adm must-gather --image=quay.io/cluster-logging-dev/cluster-logging-must-gather
+```
+
+The command above will create a local directory with a dump of the cluster-logging state.
+Note that this command will only get data related to the cluster-logging part of the OpenShift cluster.
+
+You will get a dump of:
```

Comment:
Maybe "A not all-inclusive example of what is retrieved:"  Trying to not identify what is perceived as an exact  list